sources/web/datalab/polymer/test/table-details-test.html

```diff
@@ -0,0 +1,39 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../components/table-details/table-details.html">
+    <link rel="import" href="../modules/gapi-manager/gapi-manager.html">
+  </head>
+  <body>
+
+    <test-fixture id="table-details-fixture">
+      <template>
+        <table-details></table-details>
+      </template>
+    </test-fixture>
+
+    <script src="table-details-test.js"></script>
```

Comment:
I don't see a table-details-test.js file, and table-details.test does not appear in index.html. I think either those two should be fixed, or we should just delete this file for now until we actually have those tests.